### DISCUSSION TITLE
What ziggy tool does?

### DISCUSSION THREAD

**Discussion by Solarunit** [tomasklaen/uosc#856] [Q&A]
What exactly ziggy tool does?

>we include a binary tool (one for each platform), that we call to handle stuff we can't do in lua. Currently this means searching & downloading subtitles, accessing clipboard data, and in future might improve self updating, and potentially other things.

1. searching & downloading subtitles
2. accessing clipboard data
3. self updating?

So if I don't use first 2 and I don't plan to use update feature I can remove ziggy?

**Comment by hooke007** [comment]
Of course.

**Reply by Solarunit** [reply]
Good to know, I thought it's responsible for UI too.

**Reply by hooke007** [reply]
https://github.com/tomasklaen/uosc/discussions/788

**Reply by Solarunit** [reply]
I see, and looks like it's currently not used for update, thanks
